geti_sdk/deployment/deployed_model.py

```diff
@@ -326,7 +350,18 @@ def load_inference_model(
                 raise ValueError(
                     f"Tiling is not supported for domain {self._converter.domain}"
                 )
-            self._tiler = tiler_type(model=model, execution_mode="sync")
+            classifier_name = "tile_classifier"
```

Comment:
could you please add comments to this code block, not clear why it is here

Reply:
Done!